README.md

```diff
@@ -0,0 +1,52 @@
+# Bot Function Call Detection
+
+## Description
+
+This bot detects specific function calls made by a known `deployer` address. It monitors transactions for
+`Create Agent`, `Update Agent`, and `Disable Agent` function calls.
+
+## Supported Chains
+
+- Polygon
+- Any other EVM-compatible chains
+
+## Alerts
+
+### FORTA-CREATE
+- **Description**: Fired when a transaction contains a Create Agent function call by the deployer address.
+- **Severity**: Low.
+- **Type**: "info".
+- **Metadata**:
+  - `txHash`: The hash of the transaction.
+  - `from`: The address that sent the transaction.
+  - `to`: The address that received the transaction.
+  - `data`: The input data of the transaction.
+
+### FORTA-UPDATE
+- **Description**: Fired when a transaction contains an Update Agent function call by the deployer address.
+- **Severity**: Low.
+- **Type**: "info".
+- **Metadata**:
+  - `txHash`: The hash of the transaction.
+  - `from`: The address that sent the transaction.
+  - `to`: The address that received the transaction.
+  - `data`: The input data of the transaction.
+
+### FORTA-DISABLE
```

Comment:
Thanks, I am aware it wasnt technically a task but wanted to do this too since i was interested how it would work with more different transactions to monitor :)